cmd/host-profiler/dist/host-profiler-config.yaml

```diff
@@ -1,8 +1,13 @@
+extensions:
+  ddprofiling/default: 
+    api:
```

Comment:
The api section is not required in this case, as you included the trace-agent comp. The profiles will be flushed to the trace-agent. this section should only be used in OSS collector.